### PR TITLE
Promote LoadBalancerIPMode to GA

### PR DESCRIPTION
**Pull request by RyanAoh**
Update the docs about promoting LoadBalancerIPMode to GA.

xref:
[KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1860-kube-proxy-IP-node-binding)
kubernetes/kubernetes#127348

### CODE REVIEW

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -685,8 +685,7 @@ Unprefixed names are reserved for end-users.
 
 {{< feature-state feature_gate_name="LoadBalancerIPMode" >}}
 
-As a Beta feature in Kubernetes 1.30,
-a [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) 
+The [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) 
 named `LoadBalancerIPMode` allows you to set the `.status.loadBalancer.ingress.ipMode` 
 for a Service with `type` set to `LoadBalancer`.
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md:690]:
```suggestion
For a Service of `type: LoadBalancer`, a controller can set `.status.loadBalancer.ingress.ipMode`.
```

File: content/en/docs/reference/command-line-tools-reference/kube-apiserver.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/kube-apiserver.md:1]: Omit this change; the file is (semi)autogenerated and we'll update it in a separate PR.

File: content/en/docs/reference/command-line-tools-reference/kube-controller-manager.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/kube-controller-manager.md:1]: Omit this change; the file is (semi)autogenerated and we'll update it in a separate PR.

File: content/en/docs/reference/command-line-tools-reference/kube-proxy.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/kube-proxy.md:1]: Omit this change; the file is (semi)autogenerated and we'll update it in a separate PR.

File: content/en/docs/reference/command-line-tools-reference/kube-scheduler.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/kube-scheduler.md:1]: Omit this change; the file is (semi)autogenerated and we'll update it in a separate PR.

File: content/en/docs/reference/command-line-tools-reference/kubelet.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md:1]: Omit this change; the file is (semi)autogenerated and we'll update it in a separate PR.

File: content/en/docs/concepts/services-networking/service.md
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md:1]: I would change the heading too:
```diff
-#### Specifying IPMode of load balancer status {#load-balancer-ip-mode}
+#### Load balancer IP address mode {#load-balancer-ip-mode}
```

**Review by sftim — COMMENTED** [review]
Thanks!

Please see the inline feedback.

**Comment by tengqm** [comment]
Please provide a link to the upstream **source code change** in the commit message.

**Comment by RyanAoh** [comment]
@tengqm I've updated the commit message.

**Comment by tengqm** [comment]
> @tengqm I've updated the commit message.

@RyanAoh Please provide link to the source code change, not the KEP. KEPs are too wordy to track. What we want to ensure is that the upstream source code change is proposed and merged.

**Comment by RyanAoh** [comment]
@tengqm The PR of the upstream source code is kubernetes/kubernetes#127348. I've included it in both the Git commit message and the description of this PR.

File: content/en/docs/reference/command-line-tools-reference/feature-gates/load-balancer-ip-mode.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.30"
+    toVersion: "1.32"
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/load-balancer-ip-mode.md:16]: This should be '1.31', because beta spans 1.30-1.31 inclusive.

**Comment by tengqm** [comment]
/approve
/hold

Hold can be lifted when the upstream change has been merged.

**Comment by RyanAoh** [comment]
The upstream change https://github.com/kubernetes/kubernetes/pull/127348  has been merged.
/hold cancel

**Comment by niranjandarshann** [comment]
Looks good to me 
/lgtm
